dbsync_daemon.py

```diff
@@ -44,6 +45,10 @@ def main():
     print("Logging in to Mergin...")
     mc = dbsync.create_mergin_client()
 
+    if args.force:
+        dbsync.dbsync_clean(mc)
+        dbsync.dbsync_init(mc)
```

Comment:
- there should be a check that `--skip-init` and `--force` are not used both at once (as it does not make much sense)
- here, we should do just dbsync_clean() - the init will be done later (otherwise the init would be done twice which is not needed)